cpp/src/arrow/util/thread-pool-benchmark.cc

```diff
@@ -177,26 +179,14 @@ static void ThreadPoolSpawn_Customize(benchmark::internal::Benchmark* b) {
     }
   }
   b->ArgNames({"threads", "task_cost"});
+  b->UseRealTime();
+  b->Repetitions(1);
 }
 
-static const int kRepetitions = 1;
-
-BENCHMARK(BM_WorkloadCost)->Repetitions(kRepetitions)->Apply(WorkloadCost_Customize);
-
-BENCHMARK(BM_ThreadPoolSpawn)
-    ->UseRealTime()
-    ->Repetitions(kRepetitions)
-    ->Apply(ThreadPoolSpawn_Customize);
-
-BENCHMARK(BM_SerialTaskGroup)
-    ->UseRealTime()
-    ->Repetitions(kRepetitions)
-    ->Apply(WorkloadCost_Customize);
-
-BENCHMARK(BM_ThreadedTaskGroup)
-    ->UseRealTime()
-    ->Repetitions(kRepetitions)
-    ->Apply(ThreadPoolSpawn_Customize);
+BENCHMARK(WorkloadCost)->Apply(WorkloadCost_Customize);
```

Comment:
I think this is a reference benchmark as well.

Reply:
Only this one, or all of them?

Reply:
This one only.